### ISSUE TITLE
Grails 3(12) and quartz:2.0.5 not compiling or starting app

### ISSUE THREAD

**Issue by netadmin2015** [grails-plugins/grails-quartz#53]
Java 1.8
Grails 3.0.12

removed all cache files from anywhere I can think of

New project: grails create-app junk1
add     compile 'org.grails.plugins:quartz:2.0.5' to build.gradle

grails
create-job Testme88

Edit Testme88Job and put:

  1 package testme66
  2
  3 class Testme88Job {
  4     static triggers = {
  5         simple name: 'mySimpleTrigger',  startDelay:5000, repeatCount: 1
  6     }
  7
  8     def execute() {
  9          log.debug("Domain name job processing");
 10
 11     }
 12 }

All seems good.

Now 

run-app

Lots of errors, This is the start of them.

Grails 3 / quartz 2.05 / java 1.8 / groovy 2.4.5 

I am at a loss why I cannot get jobs to work.

Honestly grails 2.4.5 I do not have any issues and may not even upgrade to grails 3 as it seems to be a completely different animal.

Any ideas ?

From scratch out of the box does not seem to work at all.

   HealthIndicatorAutoConfiguration.DiskSpaceHealthIndicatorConfiguration
      - matched (OnPropertyCondition)
Exception in thread "main" org.springframework.context.ApplicationContextException: Failed to start bean 'quartzScheduler'; nested exception is org.springframework.scheduling.SchedulingException: Could not start Quartz Scheduler; nested exception is org.quartz.SchedulerConfigException: Failure occured during job recovery. [See nested exception: org.quartz.impl.jdbcjobstore.LockException: Failure obtaining db row lock: Table "QRTZ_LOCKS" not found; SQL statement:
SELECT * FROM QRTZ_LOCKS WHERE SCHED_NAME = 'quartzScheduler' AND LOCK_NAME = ? FOR UPDATE [42102-176] [See nested exception: org.h2.jdbc.JdbcSQLException: Table "QRTZ_LOCKS" not found; SQL statement:
SELECT * FROM QRTZ_LOCKS WHERE SCHED_NAME = 'quartzScheduler' AND LOCK_NAME = ? FOR UPDATE [42102-176]]]
	at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:176)
	at org.springframework.context.support.DefaultLifecycleProcessor.access$200(DefaultLifecycleProcessor.java:51)
	at org.springframework.context.support.DefaultLifecycleProcessor$LifecycleGroup.start(DefaultLifecycleProcessor.java:346)
	at org.springframework.context.support.DefaultLifecycleProcessor.startBeans(DefaultLifecycleProcessor.java:149)
	at org.springframework.context.support.DefaultLifecycleProcessor.onRefresh(DefaultLifecycleProcessor.java:112)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:772)
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.finishRefresh(EmbeddedWebApplicationContext.java:139)

**Comment by rvanderwerf** [comment]
You have to manually create the quartz tables... the plugin does not do
that for you. Download the quartz package from quartzscheduler.org and
create the tables for your DB under the doc directory. Or set
quartz.jdbcStore = false, for the RAMStore (no database).

**Comment by AlexKovynev** [comment]
RAMStore not working from 2.0.1. Pull request awaiting!:)

**Comment by netadmin2015** [comment]
Using grails, the development mode has a database that is removed after you stop-app. I guess I could put those create tables in the Bootstrap and have them run each time. I assume version 2 had all this by default and it was removed for this version of the plugin.

I am use to (spoiled) by the Grails 2 version of everything. Grails 3 in almost every situation requires handholding.  I decided to just download the jar files and not use the plugin. Found out that grails 3 does not even have a lib directory anymore.

Anyhow thanks again.
